core/bessctl.cc

```diff
@@ -344,6 +344,54 @@ static Module* create_module(const std::string& name,
   return m;
 }
 
+static void collect_tc(const bess::TrafficClass* c, int wid,
```

Comment:
Good point. There are several functions like this in worker (there's a C++-ify TODO comment. How about I send a separate pull request for this? Otherwise I can push an extra commit here just form the naming.